### PR TITLE
Implement HTTP plugin missing options

### PR DESCRIPTION
**Pull request by w3st3ry**
Related to #7

### CODE REVIEW

**Comment by w3st3ry** [comment]
I will add documentation when #13 is merged.

File: pkg/plugins/builtin/http/README.md [hunk 1]
```diff
@@ -10,12 +10,27 @@ An action of type `http` requires the following kind of configuration:
...
+    # mandatory
     url: http://example.org/user
+    # mandatory
     method: POST
+    # optional
+    timeout_seconds: 5
+    # optional
+    basic_auth:
+      user: test
+      password: test
```
Comment (nomonamo) [inline pkg/plugins/builtin/http/README.md]: use config:
  user: {{.config.basicAuth.user}}
  password: {{.config.basicAuth.password}}

File: pkg/plugins/builtin/http/README.md [hunk 2]
```diff
@@ -10,12 +10,27 @@ An action of type `http` requires the following kind of configuration:
...
+    # optional
+    deny_redirects: false
+    # optional
+    parameters:
+    - key: foo
+      value: bar
+    # optional
     headers:
     - name:  Authorization
       value: Basic {{.config.basicAuth}}
```
Comment (nomonamo) [inline pkg/plugins/builtin/http/README.md]: change to:
  name: x-request-id
  value: xxx-yyy-zzz

File: README.md [hunk 3]
```diff
@@ -361,7 +361,11 @@ Browse [builtin actions](./pkg/plugins/builtin)
 |**`http`** | Make an http request | `url`: destination for the http call, including host, path and query params
 |           || `method`: http method (GET/POST/PUT/DELETE)
 |           || `body`: a string representing the payload to be sent with the request
-|           || `headers`: a list of headers, represented as objects composed of `name` amd `value`
+|           || `headers`: a list of headers, represented as objects composed of `name` and `value`
+|           || `timeout_seconds`: an unsigned int representing a custom HTTP client timeout in seconds
+|           || `basic_auth`: a single object composed of `user` and `password` to enable HTTP basic auth
+|           || `deny_redirects`: a boolean representing the policy of redirects
+|           || `parameters`: a list of HTTP parameters, represented as objects composed of `key` and `value`
```
Comment (nomonamo) [inline README.md]: "query parameters" for clarity

File: pkg/plugins/builtin/http/README.md (hunk 1 above)
Comment (w3st3ry) [reply pkg/plugins/builtin/http/README.md]: done.

File: pkg/plugins/builtin/http/README.md (hunk 2 above)
Comment (w3st3ry) [reply pkg/plugins/builtin/http/README.md]: done.

File: README.md (hunk 3 above)
Comment (w3st3ry) [reply README.md]: done.

**Pull request merged**